src/cmd/keys.js

```diff
@@ -217,27 +238,26 @@ module.exports = class KeysCommand {
 
 		let pubKey = temp.path({ suffix: '.pub.pem' });
 		let inform = path.extname(filename).toLowerCase() === '.der' ? 'DER' : 'PEM';
-
 		const cleanup = () => fs.unlinkSync(pubKey);
 
 		return Promise.resolve()
 			.then(() => {
 				// try both private and public versions and both algorithms
-				return utilities.deferredChildProcess('openssl ' + algorithm + ' -inform ' + inform + ' -in ' + filename + ' -pubout -outform PEM -out ' + pubKey)
+				return deferredChildProcess(`openssl ${algorithm} -inform ${inform} -in "${filename}" -pubout -outform PEM -out "${pubKey}"`)
```

Comment:
There was a period of time where I didn't like template strings, I was very silly back then and this is way nicer to read, thanks for updating.